doc/source/qip-basics.rst

```diff
@@ -290,6 +290,9 @@ produces the pdf and converts it to the png format. On Mac and Linux,
 ImageMagick can be easily installed with the command conda install imagemagick if you have conda installed.
 Otherwise, please follow the installation instructions on the ImageMagick documentation.
 
+More information related to installing these packages is also available in the
+installation guide (:ref:`circuit_plot_packages`). 
+
```

Comment:
If we move all the installation guide to `installation.rst`,  you can move the rest of it and only keep your addition.

We can remove
```
The circuit image visualization requires LaTeX and ImageMagick for display ...

On windows, you need to ...
If you want to check whether all dependencies are installed, ...
```